src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs

```diff
@@ -105,9 +105,10 @@ public string GetCurrentRequestHeader(ApiEvent eventInProgress)
 
             eventInProgress.TryGetValue(MsalTelemetryBlobEventNames.ApiIdConstStrKey, out string apiId);
             eventInProgress.TryGetValue(MsalTelemetryBlobEventNames.ForceRefreshId, out string forceRefresh);
+            eventInProgress.TryGetValue(MsalTelemetryBlobEventNames.RegionDiscovered, out string regionDiscovered);
 
             return $"{TelemetryConstants.HttpTelemetrySchemaVersion2}" +
-                $"|{apiId},{ConvertFromStringToBitwise(forceRefresh)}|";
+                $"|{apiId},{ConvertFromStringToBitwise(forceRefresh)}|{regionDiscovered}";
```

Comment:
>$"|{apiId},{ConvertFromStringToBitwise(forceRefresh)}|{regionDiscovered}"; [](start = 16, length = 74)

Would be great to see a comment in the PR that shows what this looks like in Kusto.

Has the telemetry api document been updated for .NET mentioning that we reserve the second field for region info?

Reply:
Added a comment with query and result.

@henrik-me The field where I am adding data is platform_field as mentioned [here](https://identitydivision.visualstudio.com/DevEx/_git/AuthLibrariesApiReview?path=%2FTelemetry%2FMSALServerSideTelemetry.md&version=GBdev&_a=preview). Are you talking about this document?

Reply:
this is cool. will it also get sent in the last request? there is a platform_field there as well. also, maybe make a note after line 97 about what's in our platform_config now.

---
In reply to: [492208455](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2042#discussion_r492208455) [](ancestors = 492208455)

Reply:
Can you pls. provide the output blob captured in Kusto from a test request?

---
In reply to: [492843345](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2042#discussion_r492843345) [](ancestors = 492843345,492208455)

Reply:
@henrik-me The value for MsalCur looks like 2|1004,0|westus. Do you mean adding it to the comment?

Reply:
yes